origami.json

```diff
@@ -4,6 +4,14 @@
     "origamiVersion": 1,
     "support": "andrew.betts@ft.com",
     "supportStatus": "active",
+    "browserFeatures": {
+        "required": [
+            "document.querySelector",
+            "Element.prototype.classList",
+            "Array.prototype.forEach",
```

Comment:
These aren't appearing in the polyfill service request in the demos. Maybe you need to rerun obt demo?